lib/slots.ts

```diff
@@ -39,8 +42,52 @@ export const NANOSECONDS = 'slot-nanoseconds';
 // Calendar
 export const CALENDAR_ID = 'slot-calendar-identifier';
 
+interface Slots {
+  // Instant
+  [EPOCHNANOSECONDS]: bigInt.BigInteger; // number? JSBI?
```

Comment:
Definitely not `number`. Seems like `bigInt.BigInteger` is the right type, and maybe change that once JSBI is in.